packages/main/src/ColorPalette.ts

```diff
@@ -294,11 +363,13 @@ class ColorPalette extends UI5Element {
 			this.recentColorsElements[0].focus();
 			this._currentlySelected = this.recentColorsElements[0];
 		} else {
-			[...this.colors, ...this.recentColorsElements].forEach(item => {
+			[...colorItems, ...this.recentColorsElements].forEach(item => {
```

Comment:
we call this `[...colorItems, ...this.recentColorsElements]` so many times. Consider making it getter